description/Cargo.toml

```
@@ -0,0 +1,38 @@
+[package]
+name = "scale-typegen-description"
+version = "0.0.1"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+
+# define the default features:
+
+[features]
+default = ["type-example"]
+type-example = [
+    "dep:scale-value",
+    "dep:proc-macro2",
+    "dep:rand_chacha",
+    "dep:rand",
+]
+
+[dependencies]
+anyhow = { workspace = true }
+peekmore = "1.3.0"
+smallvec = { workspace = true }
+scale-info = { workspace = true }
+
+scale-value = { version = "0.13.0", optional = true }
+proc-macro2 = { version = "1.0.69", optional = true }
+rand_chacha = { version = "0.3.1", optional = true }
+rand = { version = "0.8.5", optional = true }
+
+scale-typegen = { workspace = true }
+quote.workspace = true
```

Nit; consistent formatting with the rest:

```suggestion
quote = { workspace = true }
```